src/storage/src/read/windowed.rs

```diff
@@ -75,7 +78,12 @@ where
         }
 
         let Some(num_columns) = batches.get(0).map(|b| b.len()) else {
-            return Ok(Some(Batch::new(vec![])));
+            // the reader does not yield data, a batch of empty vectors must be returned instead of 
```

Comment:
Could we add a sqlness test case for it? It should cover #1750.

Reply:
We've already have some intergation test for this case.